inst/rmarkdown/templates/xaringan/resources/ru_theme.css

```diff
@@ -0,0 +1,239 @@
+/* 
+CSS for xaringan slides in RU flavor
+Author: Joseph V. Casillas
+Updated: 10/15/2017
+*/
+
+
+/* Package defaults that we slightly tweak */
+
+.inverse {
+  background-color: #272822;
+  color: #777872;
+  text-shadow: 0 0 20px #333;
+}
+.inverse h1, .inverse h2, .inverse h3 {
+  color: #f3f3f3;
+  line-height: 1.0em;
+}
+
+
+
+
+
+
+/*~~~~~~~~~~~~~*/
+/* Title slide */
+/*~~~~~~~~~~~~~*/
+
+/* Add image and change bg color */
+.title-slide {
+  background-image: url(http://www.jvcasillas.com/ru_xaringan/img/logo/ru_shield.png);
```

Comment:
I hope this can be `https`. Consider hosting your website on Netlify and enabling HTTPS: https://bookdown.org/yihui/blogdown/github-pages.html